src/tjenesteomrader/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Layout from 'src/layout';
+import style from './index.module.css';
+import digitalTjenesteBlob from './images/indexBlobs/digitalTjeneste.svg';
+import serviceBlob from './images/indexBlobs/strategi.svg';
+import datadrevet from './images/indexBlobs/datadrevet.svg';
+import kultur from './images/indexBlobs/kultur.svg';
+import Entrance from './components/entrance/entrance';
+
+const Tjenesteomrader = () => {
+  return (
+    <Layout>
+      <div className={style.tjenesteomrader}>
+        <h2>Våre tjenesteområder</h2>
```

Comment:
Her kan `Heading2` komponenten brukes for konsekvent styling med resten av løsningen.